### DISCUSSION TITLE
When will MySQL be available in Hasura cloud?

### DISCUSSION THREAD

**Discussion by samuel-cloete** [hasura/graphql-engine#6445] [General] [answered]
We're looking at using Hasura Cloud, but wondering when it will be available in Hasura Cloud and when the self-hosted version will be production ready? Would really help us in planning our roadmap

**Comment by tylerhannan** [comment, marked as answer]
Sorry for the delay in reply here...

We are actively working on it. Committing to a specific timeframe, at present, isn't something I'm ready for (and we'd rather provide a realistic date instead of an estimate).

But, if it helps in any way, we are targeting this quarter. Unless something drastic changes, caveats apply, etc.

**Reply by samuel-cloete** [reply]
great, thanks! that is helpful indeed!

**Reply by preetjdp** [reply]
Hi @tylerhannan Are there any updates to Hasura Support for MySQL, the last commit made to the MySQL branch was back in September 2020

https://github.com/hasura/graphql-engine/pull/5655/commits/633f084fa5b24b6f0eea71de2a33734f467811dd

Wondering what has changed

**Reply by coco98** [reply]
@preetjdp We had to pause on the MySQL work till we got the plumbing for Hasura 2.0 settled into place. Now we've started work to merge that back to the main codebase, make it feature compatible with other features. We'll likely end up opening a new PR entirely since it involves some pretty aggressive rewrites internally.

**Reply by rohit612** [reply]
We are waiting for it as well.

**Reply by snozwoz** [reply]
@coco98 Hia, any news on this? I am getting the impression MySQL is not important agenda on your roadmap

**Reply by johndevs** [reply]
Same here, we are evaluating the product for commercial purposes but the lack of communication on this roadmap item is turning us away. We need mysql support for Hasura to be a viable option for us.

**Reply by snozwoz** [reply]
not helpful to anyone else i know, but we ended up migrating our existing legacy application from MySQL to postgres, just so we could use Hasura for our new application. Loving Hasura and the DB migration wasn't too bad. Used AWS data migration service (plus a fix for auto-increment columns)
